plugins/parsers/bindata/README.md

```diff
@@ -0,0 +1,50 @@
+# BinData
+
+The "BinData" parser translates binary records consisting of multiple fields into Telegraf metrics. It supports:
+
+* Little- and Big-Endian encoding
+* bool, int8/uint8, int16/uint16, int32/uint32, int64/uint64, float32/float64 field types
+* UTF-8 and ASCII-encoded strings
+* unix, unix_ms, unix_us and unix_ns timestamp
+
+### Configuration
+
+```toml
+[[inputs.mqtt_consumer]]
+  name_override = "drone_status"
+
+  ## Data format to consume.
+  ## Each data format has its own unique set of configuration options, read
+  ## more about them here:
+  ## https://github.com/influxdata/telegraf/blob/master/docs/DATA_FORMATS_INPUT.md
+  data_format = "bindata"
+
+  ## Numeric fields endiannes, "be" or "le"
+  bindata_endiannes = "be"
+
+  ## Timestamp format - "unix", "unix_ms", "unix_us", "unix_ns"
+  bindata_time_format = "unix"
+
+  ## String encoding - "UTF-8" is the default
+  bindata_string_encoding = "UTF-8"
+
+  ## Binary data descriptor
+  ## Fields are described by:
+  ## - name - field name, use Golang naming conventions
```

Comment:
In the code it seems you assign a special meaning to the `time` field name (i.e. this is the time for the metric). Either put a big comment here that this is the case **or** better introduce a option called `bindata_time_field = "time"` allowing you to specify the field containing the metric time. If you want to be especially nice, add `bindata_drop_time_field = true` to allow the user to drop the used time field.

Reply:
I suggest to use `padding` for the case when the user wants to drop the time filed, and put that in the big comment.

Reply:
Also fine with me.